### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by dosumis**
Updating documentation links

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,12 +20,14 @@ To browse the ontology, we recommend using OLS:
 https://www.ebi.ac.uk/ols/ontologies/cl
 
 For more details on CL see:
-- https://cell-ontology.github.io/
-- [OBO Foundry](http://obofoundry.org/ontology/cl.html)
+- [https://cell-ontology.github.io/](CL Website)
```
Comment (shawntanzk) [inline README.md:23]:
```suggestion
- (CL Website)[https://cell-ontology.github.io/]
```

File: README.md [hunk 2]
```diff
@@ -20,12 +20,14 @@ To browse the ontology, we recommend using OLS:
...
+- [CL on the OBO Foundry](http://obofoundry.org/ontology/cl.html)
 
-Documentation for the Cell Ontology is also available at https://obophenotype.github.io/cell-ontology/.
 
-Editors: see the README in [src/ontology](src/ontology)
+Editors documentation: 
+* [CL Editor doc](https://obophenotype.github.io/cell-ontology/.)
+* Generic ODK workflows [src/ontology/README-editors.md](src/ontology/README-editors.md)
```
Comment (shawntanzk) [inline README.md:29]:
```suggestion
* [Generic ODK workflows](src/ontology/README-editors.md)
```

**Review by shawntanzk — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (bvarner-ebi) [inline README.md:23]:
```suggestion
- [CL Website](https://cell-ontology.github.io/)
```

File: README.md [hunk 3]
```diff
@@ -20,12 +20,14 @@ To browse the ontology, we recommend using OLS:
...
-- https://cell-ontology.github.io/
-- [OBO Foundry](http://obofoundry.org/ontology/cl.html)
+- [https://cell-ontology.github.io/](CL Website)
+- [CL on the OBO Foundry](http://obofoundry.org/ontology/cl.html)
 
-Documentation for the Cell Ontology is also available at https://obophenotype.github.io/cell-ontology/.
 
-Editors: see the README in [src/ontology](src/ontology)
+Editors documentation: 
+* [CL Editor doc](https://obophenotype.github.io/cell-ontology/.)
```
Comment (bvarner-ebi) [inline README.md:28]:
```suggestion
* [CL Editor doc](https://obophenotype.github.io/cell-ontology/)
```

**Review by bvarner-ebi — CHANGES_REQUESTED** [review]
Suggestions added

**Review by bvarner-ebi — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (dosumis) [reply README.md]: Syntax wrong.  Have fixed directly.

**Review by shawntanzk — APPROVED** [review]

**Pull request merged**